docs/source/getting_started/woodwork_types.ipynb

```diff
@@ -0,0 +1,385 @@
+    "\n",
+    "To learn more about how logical types are used in EntitySets, see the [Creating EntitySets](using_entitysets.ipynb) guide.\n",
+    "\n",
+    "To learn more about setting logical types directly on a DataFrame, see the Woodwork guide on [working with Logical Types](https://woodwork.alteryx.com/en/stable/guides/working_with_types_and_tags.html#Working-with-Logical-Types). \n",
+    "\n",
+    "## Semantic Tags\n",
+    "Semantic tags provide additional information to columns about the meaning or potential uses of data. Columns can have many or no semantic tags. Some tags are added by Woodwork, some are added by Featuretools, and users can add additional tags as they see fit.\n",
+    "\n",
+    "To learn more about setting semantic tags directly on a DataFrame, see the Woodwork guide on [working with Semantic Tags](https://woodwork.alteryx.com/en/stable/guides/working_with_types_and_tags.html#Working-with-Semantic-Tags). \n",
+    "\n",
+    "### Woodwork Standard Tags\n",
+    "\n",
+    "Woodwork will add certain semantic tags to columns at initialization. These can be standard tags that may be associated with different sets of logical types, index tags, or even tags that have some additional meaning in Woodwork.\n",
+    "\n",
+    "To get a list of the standard, index, and time index tags, you can use the `list_semantic_tags` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "11f25bd9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ft.list_semantic_tags()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29222810",
+   "metadata": {},
+   "source": [
+    "Above we see the standard tags that are defined within Woodwork. These tags inform how Featuretools is able to interpret data, but in order to generate the full set of features, Featuretools has defined a few tags of its own.\n",
+    "\n",
+    "### Semantic Tags in Featuretools\n",
+    "\n",
+    "Just like Woodwork specifies semantic tags internally, Featuretools also has certain semantic tags that have a specific meaning when they are present on a column.\n",
+    "\n",
+    "- `'last_time_index'` - added by Featuretools to the last time index column of a DataFrame. Indicates that this column has been created by Featuretools.\n",
+    "- `'foreign_key'` - used to indicate that a column has a parent column that's a primary key of another dataframe in the EntitySet. Is added by Featuretools when a Relationship is created between two DataFrames.\n",
+    "- `'date_of_birth'` - indicates that a column can be used for Primitive operations that require a date of birth. Must be added at EntitySet creation to be present.\n",
+    "\n",
+    "\n",
+    "## Woodwork Throughout Featuretools\n",
```

Comment:
This section is where this guide gets lengthy and stops being "brief", I think. 

However, given how important it is to understand the distinction between `ColumnSchemas` as type definitions vs `ColumnSchemas` as type spaces, I think this kind of description should exist _somewhere_. 

 Wondering if it might be worth having a separate guide on `Using ColumnSchemas` or `Understanding ColumnSchemas` that gets referenced here.